src/views/router_views/psc_type/psc_type.njk

```diff
@@ -35,6 +35,7 @@ summaryText: i18n.psc_type_details_title,
 text: i18n.psc_type_details_text
 }) }}
 
+<input type="hidden" name="lang" value="{{lang}}">
```

Comment:
I don't think this hidden input is needed at the moment.  On the Confirm Company screen there's a hidden input of `companyNumber` in order that the POST handling can access it again, from the request body.
But the `lang` is handled now in the request query.
I remove the `lang` input from the templates and the welsh journey seems to work.
However, when we do come to use forms maybe we'll need a input field for this.